index.css

```diff
@@ -701,3 +701,406 @@ body {
   margin-right: 20px;
 }
 
+@media screen and (min-width: 769px) {
```

Comment:
Your about section also does not match the Figma template of your choosing. Kindly fix it. Please use the same padding and margins as in the Figma design.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/177909884-9a8acd4d-5149-4454-83ce-fdac1b013a2b.png) ![image](https://user-images.githubusercontent.com/33268927/177909967-99860101-4bff-4893-866b-7b5b8f4ad979.png) | ![image](https://user-images.githubusercontent.com/33268927/177909831-379819bc-8cfa-40e7-9c11-d9a6ee6164ac.png) |